### PR TITLE
Version Module

### PR DESCRIPTION
**Pull request by jmccreight**
THIS IS NOT COMPLETE. We have yet to decide on the contents of the .version and .nwm_version files.

### CODE REVIEW

File: trunk/NDHMS/.nwm_version [hunk 1]
```diff
@@ -0,0 +1 @@
+v1.2.1+dev
```
Comment (jmccreight) [inline trunk/NDHMS/.nwm_version]: this should be blank?

File: trunk/NDHMS/utils/Makefile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Makefile 
+#
+.SUFFIXES:
```
Comment (jmccreight) [inline trunk/NDHMS/utils/Makefile]: this lline redundant/unnecessary?

File: trunk/NDHMS/utils/module_version.F [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+character (len=512), parameter :: wrfHydroVersion=WRF_HYDRO_VERSION
+character (len=512), parameter :: nwmVersion=NWM_VERSION
+
+private :: wrfHydroVersion, nwmVersion
+public  :: get_model_version, get_code_version, get_nwm_version`
+
+contains
+
+function get_model_version()
```
Comment (jmccreight) [inline trunk/NDHMS/utils/module_version.F]: this function should be removed?

File: trunk/NDHMS/.nwm_version (hunk 1 above)
Comment (kafitzgerald) [reply trunk/NDHMS/.nwm_version]: yeah, probably (until we create that branch?)

File: trunk/NDHMS/utils/module_version.F (hunk 3 above)
Comment (kafitzgerald) [reply trunk/NDHMS/utils/module_version.F]: I think it's still useful for the "title" attribute on some output files.  I use it in module_NWM_io_dict.F.

**Comment by jmccreight** [comment]
After this PR is merged, we will change the .version file and the testing to match. 
Then we will fork for V2 and give similar treatment on the branch.

**Pull request merged**